test/configs/new_cluster.conf

```diff
@@ -0,0 +1,17 @@
+# New Cluster config file
```

Comment:
In what this config file is new? I mean there is nothing in there that was not supported before. There is an nkey, but it is commented out.

Reply:
Will be adding nkey auth etc, still TBD. Will leave for now.